code/modules/paperwork/papershredder.dm

```diff
@@ -0,0 +1,173 @@
+/obj/machinery/papershredder
+	name = "Paper Shredder"
+	desc = "For those documents you don't want seen."
```

Comment:
"Disposes of papers you don't want seen, as well as IDs that are no longer needed."